### DISCUSSION TITLE
App Keywords

### DISCUSSION THREAD

**Discussion by scottrlarson** [Ulauncher/Ulauncher#993] [Q&A]
I've switched to PopOS from Windows 10 recently and I relied heavily on [Executor](https://executor.dk/). Its got some great features that could be of interest to your development of Ulauncher, such as categorizing apps into groups based on a keyword, then when you type that category's keyword you get the list of apps. I looked through the extensions but nothing stood out as a replacement for that.

My real question is, it would be great to customize the apps that are already installed in the system by being able to add other keywords to an app. For instance, if you type "activity" in the native search tool in Debian you get System Monitor. I was trying that same functionality in Ulancher and it wasn't listing that app. So before I submit a feature request. I wanted to see if anyone knew of a way to add that functionality. It looks like the shortcut's section is going to do it, but I dont see much documentation on that. Thanks for making a great app. I love the extensions support.

**Comment by friday** [comment]
We already have an issue for it #270.

It is a reasonable request, but it's not straight forward to implement given how the matching and sorting is handled now.
